i18n/locales/cn.json

```diff
@@ -287,7 +287,17 @@
     "successMessageApproved": "Submission approved and created successfully",
     "facilitySuccessfullyRejected": "Facility rejected successfully",
     "rejectSubmissionConfirmationMessage": "Are you sure you want to reject this submission?",
-    "rejectSubmissionConfirmationButton": "Yes, reject the submission"
+    "rejectSubmissionConfirmationButton": "Yes, reject the submission",
+    "placeholderTextAcceptedInsurances": "Search by insurance to add or remove an insurance",
```

Comment:
```suggestion
    "placeholderTextAcceptedInsurances": "Search by insurance",
```

Reply:
I made the suggestions here in the cn file, but they apply to all the locales. 😅